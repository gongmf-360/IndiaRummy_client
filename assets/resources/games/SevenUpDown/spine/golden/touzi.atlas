
touzi.png
size: 214,214
format: RGBA8888
filter: Linear,Linear
repeat: none
ani_flare_00000
  rotate: false
  xy: 2, 135
  size: 75, 77
  orig: 77, 78
  offset: 1, 0
  index: -1
touzi_1
  rotate: false
  xy: 66, 68
  size: 60, 65
  orig: 74, 77
  offset: 7, 6
  index: -1
touzi_2
  rotate: false
  xy: 2, 66
  size: 62, 67
  orig: 74, 77
  offset: 6, 5
  index: -1
touzi_3
  rotate: false
  xy: 79, 147
  size: 61, 65
  orig: 74, 77
  offset: 7, 6
  index: -1
touzi_4
  rotate: true
  xy: 142, 152
  size: 60, 66
  orig: 74, 77
  offset: 7, 5
  index: -1
touzi_5
  rotate: true
  xy: 2, 2
  size: 62, 65
  orig: 74, 77
  offset: 6, 6
  index: -1
touzi_6
  rotate: true
  xy: 69, 5
  size: 61, 65
  orig: 74, 77
  offset: 7, 6
  index: -1
