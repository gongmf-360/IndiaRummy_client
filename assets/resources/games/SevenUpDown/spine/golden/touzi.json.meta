{
  "ver": "1.2.3",
  "uuid": "ce2bd5ca-08e8-4cc6-91e0-53fecb55a356",
  "textures": [
    "10ecc754-e18e-484c-b3af-4a7486bc21c2"
  ],
  "scale": 1,
  "subMetas": {}
}