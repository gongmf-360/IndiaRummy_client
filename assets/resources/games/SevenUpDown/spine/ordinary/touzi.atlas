
touzi.png
size: 1024,128
format: RGBA8888
filter: Linear,Linear
repeat: none
ani_flare_00000
  rotate: false
  xy: 2, 2
  size: 102, 104
  orig: 102, 104
  offset: 0, 0
  index: -1
touzi_1
  rotate: false
  xy: 208, 6
  size: 100, 100
  orig: 100, 100
  offset: 0, 0
  index: -1
touzi_2
  rotate: false
  xy: 310, 6
  size: 100, 100
  orig: 100, 100
  offset: 0, 0
  index: -1
touzi_3
  rotate: false
  xy: 412, 6
  size: 100, 100
  orig: 100, 100
  offset: 0, 0
  index: -1
touzi_4
  rotate: false
  xy: 514, 6
  size: 100, 100
  orig: 100, 100
  offset: 0, 0
  index: -1
touzi_5
  rotate: false
  xy: 616, 6
  size: 100, 100
  orig: 100, 100
  offset: 0, 0
  index: -1
touzi_6
  rotate: false
  xy: 106, 6
  size: 100, 100
  orig: 100, 100
  offset: 0, 0
  index: -1
