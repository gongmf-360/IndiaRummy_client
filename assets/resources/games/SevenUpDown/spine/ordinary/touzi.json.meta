{
  "ver": "1.2.3",
  "uuid": "e528a532-8cce-4745-98e4-935839022cdf",
  "textures": [
    "56a4abfb-d651-420e-a489-7df1068f0df4"
  ],
  "scale": 1,
  "subMetas": {}
}