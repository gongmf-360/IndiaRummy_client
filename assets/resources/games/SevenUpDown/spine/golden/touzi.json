{"skeleton":{"hash":"lqCNjYT0CRhFVz+Xna1BUaY3S4Y","spine":"3.6.53","width":98,"height":102,"images":"./img/"},"bones":[{"name":"root"},{"name":"Effect","parent":"root"},{"name":"L-yangguang","parent":"Effect","rotation":-31.68,"x":-138.49,"y":156.18,"scaleX":1.396,"scaleY":1.396},{"name":"bg","parent":"root"},{"name":"bone","parent":"root","x":-48.31,"y":-310.93},{"name":"touzi_1","parent":"bone","x":49.49,"y":346.02},{"name":"touzi_3","parent":"touzi_1","x":9.27,"y":5.43},{"name":"touzi_4","parent":"touzi_1","x":-12.73,"y":-13.34},{"name":"touzi_5","parent":"touzi_1","x":11.71,"y":-14.83},{"name":"touzi_6","parent":"touzi_1","x":-22.77,"y":7.1},{"name":"touzi_7","parent":"touzi_1","x":-8.15,"y":-21.89}],"slots":[{"name":"bg","bone":"bg"},{"name":"ani_flare_00000","bone":"L-yangguang","blend":"additive"},{"name":"touzi_1","bone":"touzi_1","attachment":"touzi_1"},{"name":"touzi_2","bone":"touzi_3"},{"name":"touzi_3","bone":"touzi_4"},{"name":"touzi_4","bone":"touzi_5"},{"name":"touzi_5","bone":"touzi_6"},{"name":"touzi_6","bone":"touzi_7"}],"skins":{"default":{"ani_flare_00000":{"ani_flare_00000":{"width":102,"height":104}},"touzi_1":{"touzi_1":{"x":-0.44,"y":-9.89,"width":98,"height":102}},"touzi_2":{"touzi_2":{"x":-10.01,"y":-15.32,"width":98,"height":102}},"touzi_3":{"touzi_3":{"x":12.29,"y":1.88,"width":98,"height":102}},"touzi_4":{"touzi_4":{"x":-10.16,"y":4.9,"width":98,"height":102}},"touzi_5":{"touzi_5":{"x":22.32,"y":-16.08,"width":98,"height":102}},"touzi_6":{"touzi_6":{"x":7.66,"y":11.93,"width":98,"height":102}}}},"animations":{"point_1":{"bones":{"touzi_1":{"rotate":[{"time":0,"angle":0},{"time":0.0667,"angle":61.66},{"time":0.1333,"angle":129.84},{"time":0.2,"angle":-122.28},{"time":0.2667,"angle":-36.61},{"time":0.3333,"angle":0},{"time":0.4,"angle":61.66},{"time":0.5,"angle":129.84},{"time":0.5667,"angle":-122.28},{"time":0.6,"angle":-36.61},{"time":0.6667,"angle":0},{"time":0.7667,"angle":61.66},{"time":0.8333,"angle":129.84},{"time":0.9,"angle":-122.28},{"time":0.9667,"angle":-36.61},{"time":1.0333,"angle":12.98},{"time":1.1,"angle":-2.13},{"time":1.2333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0}],"translate":[{"time":0,"x":-33.76,"y":0},{"time":0.0667,"x":56.75,"y":88.77},{"time":0.1333,"x":18.12,"y":151},{"time":0.2,"x":-56.99,"y":142.42},{"time":0.2667,"x":-50.55,"y":56.58},{"time":0.3333,"x":0,"y":0},{"time":0.4,"x":56.75,"y":88.77},{"time":0.5,"x":18.12,"y":151},{"time":0.5667,"x":-56.99,"y":142.42},{"time":0.6,"x":-50.55,"y":56.58},{"time":0.6667,"x":0,"y":0},{"time":0.7667,"x":56.75,"y":88.77},{"time":0.8333,"x":18.12,"y":151},{"time":0.9,"x":-56.99,"y":142.42},{"time":0.9667,"x":-50.55,"y":56.58},{"time":1.0333,"x":-41.3,"y":5.72},{"time":1.1,"x":-39.32,"y":8.7},{"time":1.2333,"x":-33.76,"y":0,"curve":"stepped"},{"time":1.5,"x":-33.76,"y":0}],"scale":[{"time":0,"x":1,"y":1},{"time":0.1333,"x":1.189,"y":1.189},{"time":0.2667,"x":1.1,"y":1.1},{"time":0.3333,"x":1,"y":1},{"time":0.5,"x":1.189,"y":1.189},{"time":0.6,"x":1.1,"y":1.1},{"time":0.6667,"x":1,"y":1},{"time":0.8333,"x":1.189,"y":1.189},{"time":0.9667,"x":1.1,"y":1.1},{"time":1.0333,"x":1,"y":1,"curve":"stepped"},{"time":1.1,"x":1,"y":1,"curve":"stepped"},{"time":1.2333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1}]}}},"point_2":{"slots":{"touzi_1":{"attachment":[{"time":0,"name":null}]},"touzi_2":{"attachment":[{"time":0,"name":"touzi_2"}]}},"bones":{"touzi_1":{"rotate":[{"time":0,"angle":61.66},{"time":0.1,"angle":129.84},{"time":0.1667,"angle":-122.28},{"time":0.2,"angle":-36.61},{"time":0.2667,"angle":0},{"time":0.3667,"angle":61.66},{"time":0.4333,"angle":129.84},{"time":0.5,"angle":-122.28},{"time":0.5667,"angle":-36.61},{"time":0.6667,"angle":0},{"time":0.7333,"angle":61.66},{"time":0.8,"angle":129.84},{"time":0.8667,"angle":-122.28},{"time":0.9333,"angle":-36.61},{"time":1,"angle":0},{"time":1.0667,"angle":12.98},{"time":1.1667,"angle":-15.09},{"time":1.3,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0}],"translate":[{"time":0,"x":56.75,"y":88.77},{"time":0.1,"x":18.12,"y":151},{"time":0.1667,"x":-56.99,"y":142.42},{"time":0.2,"x":-50.55,"y":56.58},{"time":0.2667,"x":0,"y":0},{"time":0.3667,"x":56.75,"y":88.77},{"time":0.4333,"x":18.12,"y":151},{"time":0.5,"x":-56.99,"y":142.42},{"time":0.5667,"x":-50.55,"y":56.58},{"time":0.6667,"x":0,"y":0},{"time":0.7333,"x":56.75,"y":88.77},{"time":0.8,"x":18.12,"y":151},{"time":0.8667,"x":-56.99,"y":142.42},{"time":0.9333,"x":-50.55,"y":56.58},{"time":1,"x":-47.4,"y":0},{"time":1.0667,"x":-45.4,"y":5.72},{"time":1.1667,"x":-37.77,"y":0},{"time":1.3,"x":-35.55,"y":0,"curve":"stepped"},{"time":1.5,"x":-35.55,"y":0}],"scale":[{"time":0.1,"x":1.189,"y":1.189},{"time":0.2,"x":1.1,"y":1.1},{"time":0.2667,"x":1,"y":1},{"time":0.4333,"x":1.189,"y":1.189},{"time":0.5667,"x":1.1,"y":1.1},{"time":0.6667,"x":1,"y":1},{"time":0.8,"x":1.189,"y":1.189},{"time":0.9333,"x":1.1,"y":1.1},{"time":1,"x":1,"y":1,"curve":"stepped"},{"time":1.0667,"x":1,"y":1,"curve":"stepped"},{"time":1.1667,"x":1,"y":1,"curve":"stepped"},{"time":1.3,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1}]}}},"point_3":{"slots":{"touzi_1":{"attachment":[{"time":0.7667,"name":null}]},"touzi_3":{"attachment":[{"time":0,"name":"touzi_3"}]}},"bones":{"touzi_1":{"rotate":[{"time":0,"angle":-36.61},{"time":0.0667,"angle":0},{"time":0.1333,"angle":61.66},{"time":0.2333,"angle":129.84},{"time":0.3,"angle":-122.28},{"time":0.3333,"angle":-36.61},{"time":0.4,"angle":0},{"time":0.5,"angle":61.66},{"time":0.5667,"angle":129.84},{"time":0.6333,"angle":-122.28},{"time":0.7,"angle":-36.61},{"time":0.7667,"angle":0},{"time":0.8333,"angle":61.66},{"time":0.9,"angle":129.84},{"time":0.9667,"angle":-122.28},{"time":1.0333,"angle":12.98},{"time":1.1333,"angle":-15.09},{"time":1.2667,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0}],"translate":[{"time":0,"x":-50.55,"y":56.58},{"time":0.0667,"x":0,"y":0},{"time":0.1333,"x":56.75,"y":88.77},{"time":0.2333,"x":18.12,"y":151},{"time":0.3,"x":-56.99,"y":142.42},{"time":0.3333,"x":-50.55,"y":56.58},{"time":0.4,"x":0,"y":0},{"time":0.5,"x":56.75,"y":88.77},{"time":0.5667,"x":18.12,"y":151},{"time":0.6333,"x":-56.99,"y":142.42},{"time":0.7,"x":-50.55,"y":56.58},{"time":0.7667,"x":0,"y":0},{"time":0.8333,"x":56.75,"y":88.77},{"time":0.9,"x":18.12,"y":151},{"time":0.9667,"x":-56.99,"y":142.42},{"time":1.0333,"x":-55.04,"y":5.72},{"time":1.1333,"x":-39.26,"y":0},{"time":1.2667,"x":-36.17,"y":0,"curve":"stepped"},{"time":1.5,"x":-36.17,"y":0}],"scale":[{"time":0,"x":1.1,"y":1.1},{"time":0.0667,"x":1,"y":1},{"time":0.2333,"x":1.189,"y":1.189},{"time":0.3333,"x":1.1,"y":1.1},{"time":0.4,"x":1,"y":1},{"time":0.5667,"x":1.189,"y":1.189},{"time":0.7,"x":1.1,"y":1.1},{"time":0.7667,"x":1,"y":1},{"time":0.9,"x":1.189,"y":1.189},{"time":1.0333,"x":1,"y":1,"curve":"stepped"},{"time":1.1333,"x":1,"y":1,"curve":"stepped"},{"time":1.2667,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1}]}}},"point_4":{"slots":{"touzi_1":{"attachment":[{"time":0,"name":null}]},"touzi_4":{"attachment":[{"time":0,"name":"touzi_4"}]}},"bones":{"touzi_1":{"rotate":[{"time":0,"angle":129.84},{"time":0.0667,"angle":-122.28},{"time":0.1,"angle":-36.61},{"time":0.1667,"angle":0},{"time":0.2333,"angle":61.66},{"time":0.3,"angle":129.84},{"time":0.3667,"angle":-122.28},{"time":0.4333,"angle":-36.61},{"time":0.5,"angle":0},{"time":0.5667,"angle":61.66},{"time":0.6333,"angle":129.84},{"time":0.7,"angle":-122.28},{"time":0.7667,"angle":-36.61},{"time":0.8333,"angle":0},{"time":0.9,"angle":61.66},{"time":0.9667,"angle":12.98},{"time":1.0667,"angle":-15.09},{"time":1.2,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0}],"translate":[{"time":0,"x":18.12,"y":151},{"time":0.0667,"x":-56.99,"y":142.42},{"time":0.1,"x":-50.55,"y":56.58},{"time":0.1667,"x":0,"y":0},{"time":0.2333,"x":56.75,"y":88.77},{"time":0.3,"x":18.12,"y":151},{"time":0.3667,"x":-56.99,"y":142.42},{"time":0.4333,"x":-50.55,"y":56.58},{"time":0.5,"x":0,"y":0},{"time":0.5667,"x":56.75,"y":88.77},{"time":0.6333,"x":18.12,"y":151},{"time":0.7,"x":-56.99,"y":142.42},{"time":0.7667,"x":-50.55,"y":56.58},{"time":0.8333,"x":0,"y":0},{"time":0.9,"x":53.09,"y":48.55},{"time":0.9667,"x":-55.4,"y":99.8},{"time":1.0667,"x":-38.71,"y":0},{"time":1.2,"x":-35.28,"y":0,"curve":"stepped"},{"time":1.5,"x":-35.28,"y":0}],"scale":[{"time":0,"x":1.189,"y":1.189},{"time":0.1,"x":1.1,"y":1.1},{"time":0.1667,"x":1,"y":1},{"time":0.3,"x":1.189,"y":1.189},{"time":0.4333,"x":1.1,"y":1.1},{"time":0.5,"x":1,"y":1},{"time":0.6333,"x":1.189,"y":1.189},{"time":0.7667,"x":1.1,"y":1.1},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":0.9667,"x":1,"y":1,"curve":"stepped"},{"time":1.0667,"x":1,"y":1,"curve":"stepped"},{"time":1.2,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1}]}}},"point_5":{"slots":{"touzi_1":{"attachment":[{"time":0,"name":null}]},"touzi_5":{"attachment":[{"time":0,"name":"touzi_5"}]}},"bones":{"touzi_1":{"rotate":[{"time":0,"angle":-122.28},{"time":0.0667,"angle":-36.61},{"time":0.1333,"angle":0},{"time":0.2,"angle":61.66},{"time":0.3,"angle":129.84},{"time":0.3667,"angle":-122.28},{"time":0.4,"angle":-36.61},{"time":0.4667,"angle":0},{"time":0.5667,"angle":61.66},{"time":0.6333,"angle":129.84},{"time":0.7,"angle":-36.61},{"time":0.7667,"angle":0},{"time":0.8333,"angle":61.66},{"time":0.9,"angle":129.84},{"time":0.9667,"angle":-122.28},{"time":1.0667,"angle":-15.09},{"time":1.1667,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0}],"translate":[{"time":0,"x":-56.99,"y":142.42},{"time":0.0667,"x":-50.55,"y":56.58},{"time":0.1333,"x":0,"y":0},{"time":0.2,"x":56.75,"y":88.77},{"time":0.3,"x":18.12,"y":151},{"time":0.3667,"x":-56.99,"y":142.42},{"time":0.4,"x":-50.55,"y":56.58},{"time":0.4667,"x":0,"y":0},{"time":0.5667,"x":56.75,"y":88.77},{"time":0.6333,"x":18.12,"y":151},{"time":0.7,"x":-50.55,"y":56.58},{"time":0.7667,"x":0,"y":0},{"time":0.8333,"x":56.75,"y":88.77},{"time":0.9,"x":18.12,"y":151},{"time":0.9667,"x":-56.99,"y":142.42},{"time":1.0667,"x":-41.1,"y":0},{"time":1.1667,"x":-35.82,"y":0,"curve":"stepped"},{"time":1.5,"x":-35.82,"y":0}],"scale":[{"time":0.0667,"x":1.1,"y":1.1},{"time":0.1333,"x":1,"y":1},{"time":0.3,"x":1.189,"y":1.189},{"time":0.4,"x":1.1,"y":1.1},{"time":0.4667,"x":1,"y":1},{"time":0.6333,"x":1.189,"y":1.189},{"time":0.7,"x":1.1,"y":1.1},{"time":0.7667,"x":1,"y":1},{"time":0.9,"x":1.189,"y":1.189},{"time":0.9667,"x":1,"y":1,"curve":"stepped"},{"time":1.0667,"x":1,"y":1,"curve":"stepped"},{"time":1.1667,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1}]}}},"point_6":{"slots":{"touzi_1":{"attachment":[{"time":0,"name":null}]},"touzi_6":{"attachment":[{"time":0,"name":"touzi_6"}]}},"bones":{"touzi_1":{"rotate":[{"time":0,"angle":-122.28},{"time":0.0667,"angle":-36.61},{"time":0.1333,"angle":0},{"time":0.2,"angle":61.66},{"time":0.3,"angle":129.84},{"time":0.3667,"angle":-122.28},{"time":0.4,"angle":-36.61},{"time":0.4667,"angle":0},{"time":0.5667,"angle":61.66},{"time":0.6333,"angle":129.84},{"time":0.7,"angle":-122.28},{"time":0.7667,"angle":-36.61},{"time":0.8333,"angle":0},{"time":0.9,"angle":61.66},{"time":0.9667,"angle":129.84},{"time":1.0333,"angle":12.98},{"time":1.1333,"angle":-15.09},{"time":1.2667,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0}],"translate":[{"time":0,"x":-56.99,"y":142.42},{"time":0.0667,"x":-50.55,"y":56.58},{"time":0.1333,"x":0,"y":0},{"time":0.2,"x":56.75,"y":88.77},{"time":0.3,"x":18.12,"y":151},{"time":0.3667,"x":-56.99,"y":142.42},{"time":0.4,"x":-50.55,"y":56.58},{"time":0.4667,"x":0,"y":0},{"time":0.5667,"x":56.75,"y":88.77},{"time":0.6333,"x":18.12,"y":151},{"time":0.7,"x":-56.99,"y":142.42},{"time":0.7667,"x":-50.55,"y":56.58},{"time":0.8333,"x":0,"y":0},{"time":0.9,"x":56.75,"y":88.77},{"time":0.9667,"x":18.12,"y":151},{"time":1.0333,"x":-69.28,"y":88.17},{"time":1.1333,"x":-40.01,"y":0},{"time":1.2667,"x":-35.42,"y":0,"curve":"stepped"},{"time":1.5,"x":-35.42,"y":0}],"scale":[{"time":0.0667,"x":1.1,"y":1.1},{"time":0.1333,"x":1,"y":1},{"time":0.3,"x":1.189,"y":1.189},{"time":0.4,"x":1.1,"y":1.1},{"time":0.4667,"x":1,"y":1},{"time":0.6333,"x":1.189,"y":1.189},{"time":0.7667,"x":1.1,"y":1.1},{"time":0.8333,"x":1,"y":1},{"time":0.9667,"x":1.189,"y":1.189},{"time":1.0333,"x":1,"y":1,"curve":"stepped"},{"time":1.1333,"x":1,"y":1,"curve":"stepped"},{"time":1.2667,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1}]}}}}}